app/views/courses/preview/_train_with_disabilities.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-train-with-disabilities">Training with disabilities and other needs</h2>
+  <% if @provider.train_with_disability.present? %>
+    <%= markdown(@provider.train_with_disability) %>
+  <% else %>
+    <p class="missing-section">Please add details for this section.</p>
```

Comment:
Should this also link to the about the org section?

Reply:
The links we're inconsistent in the sections so I removed them to keep it simple...

Reply:
I think the link was added here because it's not obvious that this section isn't added via the organisation page, rather than the course.